ts/services/DashboardService.ts

```diff
@@ -46,12 +47,27 @@ class DashboardService {
         logger.info(response.dashboard.name);
       }
       if (config.format === 'json') {
+        // remove some internal keys
+        this.scrubDashboardKeys(response.dashboard);
         logger.info(JSON.stringify(response, null, 2));
       }
     } catch (e) {
       logger.error('There was an error getting the dashboard: ' + e);
     }
   }
+
+  private scrubDashboardKeys(dashboard) {
```

Comment:
What do you think about changing this method to not modify the original dashboard object and instead returned a scrubbed object? I get nervous about methods which internally modify an argument and don't return a result.

Something like adding in the line `const scrubbed = Object.assign({}, dashboard);` then modifying and deleting that scrubbed object.

Reply:
meh, it's modifying an inline dashboard to be committed to source, I see what you are saying but it's also possible this becomes a moot point if we can get api to scrub response properly as well.

Reply:
Hmm, I'm now looking at this PR and realizing that this is the full dashboard service. I don't think the IDs should be scrubbed from here because scrubbing is only useful for package exports. The CLI for getting dashboard JSON is independent of those packages.

Reply:
My use case: 
- I modify my dashboard (which is a copy remind you) 
- I use `metricly dashboard get <id> > dashboard/my.awesome.dashboard.json` 
with that I get a bunch of irrelevant ids.

Reply:
How do you get the dashboard ID? You scrub it in line 60 which means the list command renders an undefined.

Reply:
😢you know this would be a good place for tests 😆 
OK I suppose we need `package export ...` that does any scrubbing, thinking out loud, what about: 
- `package export dashboards --location ./dashboards/ --filter "^EXP.*"`  
- or `package export dashboard <dbid>` may be first/simpler?

Reply:
I'm thinking more along the lines of the latter. We could make an export config as well and store the IDs in a dot file locally. Something like:
```
$ package export config
$ Which dashboards do you want to export (comma delimited list of IDs)? xxx,yyy,zzz
$ Which policies do you want to export (comma delimited list of IDs)? xxx,yyy,zzz
$ cat .pkg.json
{
  "dashboards": ["xxx", "yyy", "zzz"],
  "policies": ["xxx", "yyy", "zzz"]
}
```

Then to re-export your package you just run `package export` and it pulls those dashboards and policies down, scrubs them, and you're g2g.